problem-solver/cxx/format-translators-module/agent/key-elements-orderer-agent/dummy_key_elements_orderer_agent.cpp

```diff
@@ -0,0 +1,81 @@
+#include "dummy_key_elements_orderer_agent.hpp"
+
+#include "keynodes/format_translators_keynodes.hpp"
+
+namespace formatTranslators
```

Comment:
I think this namespace is extra

Reply:
Yes, they may appear, but even if they do, nothing is broken because we have fixed version of sc-machine

Reply:
I am pro-namespaces
there is already too many reasons why students can't successfully build projects